app/assets/stylesheets/style.css.scss

```diff
@@ -99,6 +99,12 @@ fieldset.tagline {
 
 #neighborhood_list img {
   border-radius: 10px;
+  opacity: .67;
+  filter: alpha(opacity=67);
+}
+
+#neighborhood_list a{
```

Comment:
Avoid using id selectors<br>Merge rule `#neighborhood_list a` with rule on line 96<br>Selector `neighborhood_list` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space